README.md

```diff
@@ -14,13 +14,32 @@ TODO
 
 \subsubsection*{Cantor set}
 \subsubsection*{Cartesian space ${\mathbb R}^n$}
+The set of all $n$-tuples, $\{ (x_1, x_2, ... , x_n) : x_i \in \mathbb{R} \}$.\\
+The number $n$ is called the dimension of $\mathbb{R}^n$, and the $n$-tuples
+can be thought of as vectors, since they form a natural vector space.\\
+The set is typically augmented with the "Usual Topology", or sometimes the
+"Euclidean Topology", whose basis is the set of all balls,
+$\{ (x_1, x_2, ..., x_n) : \sqrt{x_1^2 + x_2^2 + ... + x_n^2} < r, r > 0 \}$.
+Equivalently, the topology can be defined so that a subset $U$ is open if and only
+if for all $u\in U$, there is a ball $B_r$ so that $u \in B_r \subseteq U$.
+If no topology on the space is mentioned it is assumed to be the Usual.
```

Comment:
Why Usual in caps?

Reply:
To specifically call back to the technical definition.  Its just for extra emphasis.